homeassistant/components/device_tracker/traccar.py

```diff
@@ -39,6 +40,7 @@
     vol.Optional(CONF_PORT, default=8082): cv.port,
     vol.Optional(CONF_SSL, default=False): cv.boolean,
     vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
+    vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL): cv.time_period,
```

Comment:
Remove this. It's alreadyppart of the base schema. Instead define the constant `SCAN_INTERVAL` at the module level and set it to default update interval.

Reply:
Thanks, I didn't know that